docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,810 @@ thor:
-          resource manifest.</para>
+          <para>Each Roxie must be told to trust the other. In the hpcc2
+          values file the Roxie service has its
+          <literal>trustClients</literal> array set to include hpcc1.roxie,
+          and likewise the hpcc1 values file has its Roxie service
+          <literal>trustClients</literal> array set to include hpcc2.roxie.
+          These files can be found in the HPCC-Platform repository at
+          helm/examples/vault-pki-remote/values-hpcc2.yaml and
+          helm/examples/vault-pki-remote/values-hpcc1.yaml. Here are the
+          relevant parts of the hpcc2 values file:</para>
 
-          <example>
-            <title>Remote Issuer with Alternative Domains</title>
+          <programlisting>            roxie:
+            - name: roxie2
+              disabled: false
+              services:
+              - name: roxie2
+                servicePort: 29876
+                visibility: local
+                trustClients:
+                - commonName: roxie1.hpcc1        </programlisting>
 
-            <programlisting>
-certificates:
-  enabled: true
-  issuers:
-    remote:
-      name: remote-issuer
-      kind: ClusterIssuer
-      domain: example.com
-      alternativeDomains:
-        - subdomain1.example.com
-        - subdomain2.example.com
-        - api.example.org
-        - secure.mycompany.net
-          </programlisting>
-          </example>
+          <sect5 id="InstallHashicorpVault">
+            <title>Install Hashicorp Vault Service</title>
 
-          <para>In this configuration:</para>
+            <para>This scenario is for development only, you would never want
+            to deploy this way in a real world production environment.
+            Deploying in dev mode sets up an in memory kv store that won't
+            persist secret values across restart, and the vault will
+            automatically be unsealed.</para>
 
-          <itemizedlist>
-            <listitem>
-              <para>The primary domain is
-              <literal>example.com</literal></para>
-            </listitem>
+            <para>In dev mode the default root token is simply the string
+            "root".</para>
 
-            <listitem>
-              <para>Certificates will also be valid for all domains listed in
-              <literal>alternativeDomains</literal></para>
-            </listitem>
+            <para>Add Hashicorp Helm repository:</para>
 
-            <listitem>
-              <para>The generated Certificate resource will include all
-              domains in its <literal>dnsNames</literal> section</para>
-            </listitem>
-          </itemizedlist>
-        </sect4>
-      </sect3>
+            <programlisting>helm repo add hashicorp https://helm.releases.hashicorp.com</programlisting>
 
-      <sect3 id="Certificate_Generation_Process">
-        <title>Certificate Generation Process</title>
+            <para>Update Helm repositories.</para>
```

Comment:
minor: remove this para text and collapse the associated programlisting into the previous one.